File: src/entity/enemy/DurableEnemy.java
package entity.enemy;

import javafx.scene.image.Image;

public class DurableEnemy extends Enemy {
	
	public static final String IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile247.png";
	
	public DurableEnemy(int x, int y) {
		super(x, y, 25, .5);
		img = new Image(IMAGE_URL);
		setDir('E');
	}
	
}

File: src/entity/enemy/DurableEnemy.kt
package entity.enemy

import javafx.scene.image.Image

class DurableEnemy(x: Int, y: Int) : Enemy(x.toDouble(), y.toDouble(), 25, .5) {
    override var img: Image = Image(IMAGE_URL)

    init {
        setDir('E')
        reward = 3
    }

    companion object {

        val IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile247.png"
    }

}
